samples/notesync/bootconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "remoteAccessConsumerKey": "3MVG9Iu66FKeHhIMmUBkEV8o6PfICh5xUnN89HyifhHcvSSBM3WUYyG9buZBezdrhdYilqlXHHbqamC5MqaKP", // mobile2
```

Comment:
We should take that out (the mobile2.t client id) when we released that sample app.